recipes/pyscf-dispersion/meta.yaml

```diff
@@ -0,0 +1,66 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 16 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "pyscf-dispersion" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+source:
+  url: https://github.com/pyscf/dispersion/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a4ec720a27ce269df459409fd89bb6e9e725889c16339a8cdec1739dc36f8da6
+build:
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ stdlib("c") }}
+    - cmake >=3.26
+    - make
+    - python
+  host:
+    - libblas
+    - liblapack
+    - python
+    - pip
+    - setuptools
+    - pyscf >=2.6
+  run:
+    - python
+    - pyscf >=2.6
+test:
+  imports:
+    - pyscf
+  requires:
+    - pip
+    - pyscf >=2.6
```

Comment:
This should not be necessary.
```suggestion
```

Reply:
It is updated with more accurate tests.

Reply:
You should not need this depedency here, it's already in run.